web/modules/custom/collection/collection.behat.inc

```diff
@@ -157,6 +167,36 @@ class CollectionSubContext extends DrupalSubContextBase implements DrupalSubCont
   }
 
   /**
+   * Indexes items of entity type: collection.
+   *
+   * In addiction, views cache data are being cleared.
+   *
+   * @throws \Exception
+   *   When no items are indexed.
```

Comment:
This is not really correct, the exception is thrown when the index doesn't exist, not when there are no items to index.